monkey/monkey_island/linux/install_mongo.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+export os_version_monkey=$(cat /etc/issue)
+MONGODB_DIR=$1 # If using deb, this should be: /var/monkey/monkey_island/bin/mongodb
+
+if  [[ $os_version_monkey == "Ubuntu 14.04"* ]] ;
```

Comment:
Do we even support Ubuntu 14.04? The python version there is too old for Tornado which I think we require on the Island.

Reply:
fixed